src/concept/mod.rs

```diff
@@ -44,6 +44,22 @@ pub enum Concept {
     Attribute(Attribute),
 }
 
+impl Concept {
+    pub fn type_label_cloned(&self) -> String {
+        // FIXME: Add a Label type to simplify this function
+        match self {
+            Concept::RoleType(RoleType { label, .. }) => format!("{label}"),
+            Concept::RootThingType(_) => RootThingType::LABEL.to_owned(),
+            Concept::EntityType(type_) => type_.label.clone(),
+            Concept::RelationType(RelationType { label, .. }) => label.clone(),
+            Concept::AttributeType(AttributeType { label, .. }) => label.clone(),
+            Concept::Entity(Entity { type_: EntityType { label, .. }, .. }) => label.clone(),
+            Concept::Relation(Relation { type_: RelationType { label, .. }, .. }) => label.clone(),
+            Concept::Attribute(Attribute { type_: AttributeType { label, .. }, .. }) => label.clone(),
```

Comment:
can we reorder these so that `RootThingType` comes first, then all the `Type` in order `Entity, Relation, Role, Attribute` , then Entity/Relation/Attribute

Reply:
We can, of course. Now this order corresponds to the order of the `Concept` variants. 
@dmitrii-ubskii Let's reorder?

Reply:
`Role` should definitely not be the first one, I'd say make both of them in the order I proposed :)